app/components/dashboard/DcSunburstChart.vue

```
@@ -0,0 +1,598 @@
+    default: 500 // ミリ秒単位の長押し判定の閾値
+  },
+  margins: {
+    type: [Object, Number],
+    default: null
+  }
+});
+
+// イベント定義
+const emit = defineEmits(['filtered']);
+
+let chart;
```

ESLint

[error] Extra semicolon.

(@stylistic/semi)